packages/volto/src/helpers/Api/Api.js

```diff
@@ -23,14 +26,16 @@ export function formatUrl(path) {
 
   if (path.startsWith('http://') || path.startsWith('https://')) return path;
 
-  const adjustedPath = path[0] !== '/' ? `/${path}` : path;
   let apiPath = '';
   if (settings.internalApiPath && __SERVER__) {
     apiPath = settings.internalApiPath;
   } else if (settings.apiPath) {
     apiPath = settings.apiPath;
   }
 
+  let adjustedPath = path[0] !== '/' ? `/${path}` : path;
+  adjustedPath = stripPrefixPath(adjustedPath);
```

Comment:
I have doubts about the `stripPrefixPath`... if the deployment is configured also with the path... then the URLs of the backend will match the frontend config, then there will be no need of stripping anything, right? Or I am missing something? True, that then the deployment will be the same as if there will be no prefix, but I don't know if not doing that has any other hidden drawback...

/cc @ericof @davisagli